src/test/article.spec.js

```diff
@@ -1199,3 +1203,92 @@ describe('GET /api/v1/articles/popular', () => {
     findAllStub.restore();
   });
 });
+describe('POST api/v1/:articleId/comments/:commentId/like', () => {
+  beforeEach(async () => {
+    await db.Article.destroy({ cascade: true, truncate: true });
+    await db.User.destroy({ cascade: true, truncate: true });
+    // await db.Commentlike.destroy({ cascade: true, truncate: true });
+  });
+
+  it('should like a comment', async () => {
+    const author = await createUser();
+    const commenter = await createUser();
+    const liker = await createUser();
+
+    const createdArticle = await createArticle(
+      await generateArticle({ authorId: author.id })
+    );
+
+    const createdComment = await createComment(
+      createdArticle.id,
+      await generateComment({ userId: commenter.id })
+    );
+    const { id: commentId } = createdComment[0].dataValues;
+
+    const token = helper.jwtSigner({ id: liker.id });
+    const res = await server()
+      .post(`/api/v1/${createdArticle.id}/comments/${commentId}/like`)
+      .set('token', token)
+      .send({ type: 'comment' });
+    expect(res.status).to.equal(201);
+    expect(res.body.data).to.be.equal('Comment liked');
+  });
+
+  it('should throw error if article id is invalid', async () => {
```

Comment:
This test should not depend on the previous, please create a separate user or this test

Reply:
kindly check below to see the user creating the like